app/src/main/java/com/mapbox/android/gestures/testapp/HelpDialogFragment.java

```diff
@@ -0,0 +1,35 @@
+package com.mapbox.android.gestures.testapp;
+
+import android.app.DialogFragment;
+import android.os.Bundle;
+import android.view.LayoutInflater;
+import android.view.View;
+import android.view.ViewGroup;
+import android.widget.Button;
+
+public class HelpDialogFragment extends DialogFragment {
+  public static final String TAG = "help_dialog_fragment_tag";
```

Comment:
can be private?

Reply:
Used in the Activity, can be package private.